app.py

```diff
@@ -0,0 +1,67 @@
+from flask import Flask
+import requests
+import pandas as pd
+from requests.sessions import Request
+import numpy as np
+from flask import Flask, url_for
+from flask import Flask, request, render_template, redirect
+import json
+import pymongo
+
+
+
+app = Flask(__name__)
+
+
+client = pymongo.MongoClient("mongodb+srv://base_user:base_user_password@cluster0.dbcb9.mongodb.net/first")
+db = client.first
+collection = db.berkay

```

Comment:
you can change this variable name because it may cause a conflict. recommended name is something like market_collection

Reply:
I think my file organization can be problematic as well. I will change it and create a new pull request.